index.rst

```diff
@@ -1,11 +1,11 @@
 MoveIt! Tutorials
 =================
 
-These tutorials will run you through how to use MoveIt! with your robot. It is assumed that you have already configured MoveIt! for your robot - check the `list of robots running MoveIt! <http://moveit.ros.org/robots/>`_ to see whether MoveIt! is already available for your robot. Otherwise, skip to the tutorial on Setting up MoveIt! for your robot. If you just want to test MoveIt!, use the Panda as your quick-start robot.
+These tutorials will step you through using MoveIt! with your robot. It is assumed the robot you are working with has already been configured to work with your robot - check the `list of robots running MoveIt! <http://moveit.ros.org/robots/>`_ to see whether MoveIt! is already available for your robot. Otherwise, skip to the `Integration with a New Robot<#integration-with-a-new-robot>`_ tutorials, below. If you just want to quickly try MoveIt!, use the default Panda robot as your quick-start demo.
```

Comment:
"These tutorials will step you through using MoveIt! with your robot. It is assumed the robot you are working with has already been configured to work with your robot"